src/App.tsx

```diff
@@ -15,36 +17,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  SortAlphabetically = 'Sort alphabetically',
+  SortByLength = 'Sort by length',
+  Reverse = 'Reverse',
+  Reset = 'Reset',
+}
```

Comment:
technically it is not sort type. it is just texts for buttons

instead better to make enum SortType or SortBy for 
const SORT_FIELD_BY_NAME = 'name';
const SORT_FIELD_BY_LENGTH = 'length';
```
enum SortBy {
   None: 'none',
   ByName: 'name',
   ByLength: 'length',
}
```
and use everywhere.